lib/node_modules/@stdlib/complex/float32/base/identity/src/main.c

```diff
@@ -0,0 +1,33 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2021 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/complex/float32/base/identity.h"
+#include <complex.h>
+
+/**
+* Evaluates the identity function for a single-precision complex floating-point number.
+*
+* @param z       number
+* @return        input value
+*
+* @example
+* float complex y = stdlib_base_complex64_identity( 3.0f+3.0f*I );
+*/
+float complex stdlib_base_complex64_identity( const float complex z ) {
```

Comment:
Note to self: we should be using `stdlib_complex64_t`. This package needs updating after merging.